app/assets/javascripts/controllers/mixins/charts-mixin.js

```diff
@@ -1,4 +1,4 @@
-angular.module('miq.util').factory('containerChartsMixin', ['pfUtils', function(pfUtils) {
```

Comment:
So we are using this file over `ems_common/charts-mixin`, that correct? because I noticed that this file is missing few configs that were present on the older.

Reply:
`charts-mixin` should be preferred. If this PR removed chartsMixin and replaced it with an older version, that's a mistake.

Reply:
You are right @himdel, I've heard about plans to unify the container with the other providers, so I won't touch it :sweat_smile:, will just rollback the `chartsMixin`

Reply:
Thanks @felipedf .. feel free to touch it enough so that you don't see any more breakages ;)

I'd love to unify these more, but.. obviously not breaking stuff is more important :)

Reply:
Ok @himdel will try to help whenever I can (:

Reply:
@felipedf apologies if during all the back and forth rebasing i missed/lost some piece of code, let me know i will be happy to help with recovering those.

Reply:
@h-kataria it is fine :smile: 
I got it all working just fine now.. I was just lost after pulling from master and ending up with 3 `charts-mixin` in my local.. i was like LOL :sweat_smile: